contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -4,48 +4,55 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"io"
 	"log/slog"
-	"net"
 	"net/http"
 	"os"
 	"path/filepath"
-	"strings"
 	"time"
 
-	"github.com/charmbracelet/lipgloss"
-	"github.com/fsnotify/fsnotify"
-	"github.com/gnolang/gno/contribs/gnodev/pkg/dev"
 	gnodev "github.com/gnolang/gno/contribs/gnodev/pkg/dev"
 	"github.com/gnolang/gno/contribs/gnodev/pkg/emitter"
-	"github.com/gnolang/gno/contribs/gnodev/pkg/logger"
 	"github.com/gnolang/gno/contribs/gnodev/pkg/rawterm"
 	"github.com/gnolang/gno/contribs/gnodev/pkg/watcher"
-	"github.com/gnolang/gno/gno.land/pkg/gnoweb"
-	zaplog "github.com/gnolang/gno/gno.land/pkg/log"
+	"github.com/gnolang/gno/gno.land/pkg/integration"
 	"github.com/gnolang/gno/gnovm/pkg/gnoenv"
-	"github.com/gnolang/gno/gnovm/pkg/gnomod"
 	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/gnolang/gno/tm2/pkg/crypto"
+	"github.com/gnolang/gno/tm2/pkg/crypto/keys"
 	osm "github.com/gnolang/gno/tm2/pkg/os"
-	"github.com/muesli/termenv"
 )
 
 const (
 	NodeLogName        = "Node"
 	WebLogName         = "GnoWeb"
 	KeyPressLogName    = "KeyPress"
 	EventServerLogName = "Event"
+	AccountsLogName    = "Accounts"
+)
+
+var (
+	DefaultDeployerName    = integration.DefaultAccount_Name
+	DefaultDeployerAddress = crypto.MustAddressFromString(integration.DefaultAccount_Address)
+	DefaultDeployerSeed    = integration.DefaultAccount_Seed
 )
 
 type devCfg struct {
+	// Listeners
 	webListenerAddr          string
 	nodeRPCListenerAddr      string
 	nodeP2PListenerAddr      string
 	nodeProxyAppListenerAddr string
 
+	// Users default
+	deployKey          string
+	home               string
+	root               string
+	additionalAccounts varAccounts
+	balancesFile       string
```

Comment:
Do you think we should premine the balance sheet from `genesis_balances.txt`?

Reply:
No, I personally don't think it should depend on `genesis_balances.txt`, as the `genesis_txs.jsonl` file isn't loaded anyway. Keeping the node as pure as possible (beside loading `example` folder). 
Users can explicitly specify those if they really want to anyway.